api/types.go

```diff
@@ -20,3 +24,17 @@ type ProcessResponse struct {
 	Nonce            uint64          `json:"nonce,omitempty"`
 	Address          string          `json:"address,omitempty"`
 }
+
+// Vote is the struct to represent a vote in the system. It will be provided by
+// the user to cast a vote in a process.
+type Vote struct {
+	ProcessID        types.HexBytes      `json:"processId"`
+	Commitment       types.HexBytes      `json:"commitment"`
+	Nullifier        types.HexBytes      `json:"nullifier"`
+	Cipherfields     elgamal.Ciphertexts `json:"cipherfields"`
+	CensusProof      types.CensusProof   `json:"censusProof"`
+	BallotProof      parser.CircomProof  `json:"ballotProof"`
+	BallotInputsHash types.HexBytes      `json:"ballotInputsHash"`
+	PublicKey        types.HexBytes      `json:"publicKey"`
```

Comment:
I'd not include PublicKey here, the Sequencer must take it from the State instead.

Reply:
Its public key of the user